code/_helpers/unsorted.dm

```diff
@@ -770,6 +770,15 @@ var/global/list/WALLITEMS = list(
 			colour += temp_col
 	return "#[colour]"
 
+/proc/refresh_lobby_browsers()
+	for(var/mob/new_player/player in global.player_list)
+		INVOKE_ASYNC(global.using_map, /datum/map/proc/show_titlescreen, player.client)
+		INVOKE_ASYNC(player, /mob/new_player/proc/show_lobby_menu)
```

Comment:
Why are these async?

Reply:
First one ported from Tau and I pretty sure it's to not cause problems during updating lobby screen.

Reply:
Changed according to Lohikar's tips.